scripts/2.TCGA-MLexample-covariates.py

```diff
@@ -281,26 +284,33 @@ def get_threshold_metrics(y_true, y_pred):
 }
 
 
-# In[22]:
-
-def get_coefficients(classifier, X_mat):
-    coef_df = pd.DataFrame.from_items([
-        ('feature', X_mat.columns),
-        ('weight', classifier.coef_[0]),
-    ])
+# In[19]:
 
+def get_coefficients(classifier, feature_set):  
```

Comment:
Perhaps we should move this function to `utils.py`? This functionality should really be in `sklearn`... not sure most users really need to see the details.